Todo.Web/Server/Program.cs

```diff
@@ -40,7 +40,7 @@
 }
 
 app.UseHttpsRedirection();
-app.UseStaticFiles();
+app.MapStaticAssets();
```

Comment:
Seeing the Map next to the Use* triggers my ocd

Reply:
You are right, there a warning showing when use `UseStaticFiles()` with .NET9 

![image](https://github.com/user-attachments/assets/a8669228-30b0-4100-9d9d-c62cc48f5138)

Also, Microsoft Docs explain with not best ordering like: 

https://learn.microsoft.com/en-us/aspnet/core/fundamentals/static-files?view=aspnetcore-9.0#serve-files-in-web-root